watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java

```diff
@@ -63,6 +65,7 @@ public void configureViewWithMiniSite(View view, MiniSite miniSite) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setHasOptionsMenu(true);
```

Comment:
Do we need to do this if we're just gonna use an empty menu? Would it work if we said

<code> setHasOptionsMenu(false); </code>